src/Microsoft.DotNet.ProjectJsonMigration/transforms/IncludeContextTransform.cs

```diff
@@ -25,6 +25,16 @@ internal class IncludeContextTransform : ConditionalTransform<IncludeContext, IE
                         && includeContext.IncludeFiles != null 
                         && includeContext.IncludeFiles.Count > 0);
 
+        private bool IsPatternBlackListed(string pattern)
+        {
+            if (_patternsBlackList == null)
```

Comment:
Instead of having to do these null checks all over, can you simply assign an empty list to it if the parameter is null? This way you can just do the Contains.